core/controllers/questions_list.py

```diff
@@ -61,12 +65,18 @@ def get(self, comma_separated_skill_ids):
         except Exception as e:
             raise self.PageNotFoundException(e)
 
-        (
-            question_summaries, merged_question_skill_links,
-            next_start_cursor) = (
-                question_services.get_displayable_question_skill_link_details(
-                    constants.NUM_QUESTIONS_PER_PAGE, skill_ids, start_cursor)
-            )
+        question_summaries, merged_question_skill_links, _ = (
+            question_services.get_displayable_question_skill_link_details(
+                constants.NUM_QUESTIONS_PER_PAGE + 1, skill_ids, offset)
+        )
+
+        if len(question_summaries) <= constants.NUM_QUESTIONS_PER_PAGE:
+            more = False
+        else:
+            more = True
```

Comment:
Also, I feel like this calculation and the return of `more` should be done inside the relevant function itself. So, instead of the function returning `next_cursor`, it returns a boolean `more`, what do you think?

Reply:
I think the requirement of more is only in our get function and we should not do this. But I can surely change it. Please let me know what should I do. Thanks!

Reply:
There was no need of more. PTAL. Changed.